source/openapi-spec.yml

```diff
@@ -575,6 +595,39 @@ components:
           type: string
           description: The candidate’s ethnicity as a two-digit HESA code (https://www.hesa.ac.uk/collection/c19053/e/ethnic)
           example: "10"
+    Attribution:
+      type: object
+      additionalProperties: false
+      required:
+        - full_name
+        - email
+        - user_id
+      properties:
+        full_name:
+          type: string
+          description: The full name of the user who made this update
+          example: "Jane Smith"
+        email:
+          type: string
+          description: The email address of the user who made this update
+          example: "jane.smith@provider.ac.uk"
+        user_id:
+          type: string
+          description: The ID of the user in the Student Record System, used for disambiguation
+          example: "12345"
+    MetadataForUpdate:
```

Comment:
Possibly a naive question: why do not use just the `attribution` field, instead of creating `MetadataForUpdate` (that does not explicitly say what it contains) 
The `attribution` could contain the content and timestamp within.

Reply:
To my mind `attribution` and `timestamp` are separate concepts

Reply:
I think your point about `MetadataForUpdate` is a good one — I've renamed it `TimestampAndAttribution`